_data/data.yaml

```diff
@@ -64,7 +64,7 @@ tabs:
             source: 'https://www.donaldjtrump.com/press-releases/donald-j.-trump-delivers-groundbreaking-contract-for-the-american-vote1'
           -
             text: 'Donald Trump will refuse the $400,000 annual president''s salary or only take $1'
-            status: notStarted
+            status: Complete
```

Comment:
You're correct, it seems as though it hasn't been confirmed. Although it seems he may end up taking $1 for his salary.